operate/webapp/src/main/java/io/camunda/operate/webapp/security/SessionRepository.java

```diff
@@ -20,6 +20,9 @@
 import java.util.Optional;
 
 public interface SessionRepository {
+
+  String POLLING_HEADER = "x-is-polling";
```

Comment:
You can make this static

Reply:
I think in interfaces the fields are declared as public static final by default.
I use the IntelliJ plugin [SonarLint](https://plugins.jetbrains.com/plugin/7973-sonarlint) to check the code and it also says: 
`Modifier 'static' is redundant for interface fields`